### PR TITLE
doc: update osd-class-path error messages

### PR DESCRIPTION
**Pull request by TheJJ**
the return value is "95 Operation not supported"

This hopefully helps people googling for the error message :)

Signed-off-by: Jonas Jelten <jj@sft.lol>

## Checklist
- [x] Updates documentation if necessary

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by tchaikov — APPROVED** [review]

File: doc/dev/osd-class-path.rst [hunk 1]
```diff
@@ -4,13 +4,25 @@
...
+  2021-03-16 01:26:59.013 7f6c6dff3640 10 _load_class rbd from /usr/local/lib64/rados-classes/libcls_rbd.so
+  2021-03-16 01:26:59.013 7f6c6dff3640  0 _load_class could not stat class /usr/local/lib64/rados-classes/libcls_rbd.so: (2) No such file or directory
+  2021-03-16 01:26:59.013 7f6c6dff3640 -1 osd.3 112 class rbd open got (2) No such file or directory
+  2021-03-16 01:26:59.013 7f6c6dff3640  1 -- [...] --> [...] -- osd_op_reply(5 rbd_directory [call rbd.dir_add_image] v0'0 uv0 ondisk = -95 ((95) Operation not supported)) v8 -- 0x7f6c6800fed0 con 0x7f6cb80100c0
+
+This means the OSD could not find ``libcls_rbd.so``.
+You can customize the load path of these modules in ``ceph.conf`` with ``osd_class_dir``.
+By default, this is ``$libdir/rados-classes``, so when developing, you likely have to adjust the path.
+
+These class libraries are used for extending RADOS, see :doc:`../architecture`.
```
Comment (tchaikov) [inline doc/dev/osd-class-path.rst:28]: nit, would be better to point to https://docs.ceph.com/en/latest/rados/api/objclass-sdk/. ideally, we could add a label right before the section header in objclass-sdk.rst, like

```
.. _`sdk-for-ceph-object-classes`:
```

and reference it like
```
see :ref:`sdk-for-ceph-object-classes`.
```

File: doc/rados/api/objclass-sdk.rst [hunk 2]
```diff
@@ -2,6 +2,8 @@
 SDK for Ceph Object Classes
 ===========================
 
+.. _`rados-objclass-api-sdk`:
```
Comment (tchaikov) [inline doc/rados/api/objclass-sdk.rst:5]: might want to put the label *before* the section header. something like
```rst
.. _`rados-objclass-api-sdk`:

SDK for Ceph Object Classes
===========================
```

Comment (tchaikov) [reply doc/rados/api/objclass-sdk.rst:5]: see https://ceph--40152.org.readthedocs.build/en/40152/dev/osd-class-path/ for the rendered page.

Comment (TheJJ) [reply doc/rados/api/objclass-sdk.rst:5]: uh kay sorry :)

**Review by tchaikov — APPROVED** [review]

File: doc/rados/api/objclass-sdk.rst (hunk 2 above)
Comment (tchaikov) [reply doc/rados/api/objclass-sdk.rst]: thank you! it's perfect now!

**Review by zdover23 — APPROVED** [review]

**Pull request merged**